tox.ini

```
@@ -25,6 +28,11 @@ commands=
     yamllint -s .
 
 [testenv:docs]
+description = Build docs
 deps = -r{toxinidir}/docs/requirements.txt
```

(since you have alabaster/sphinx in the constraints)
```suggestion
deps = -r{toxinidir}/docs/requirements.txt -c{toxinidir}/test/constraints.txt
```